src/TMainConsole.cpp

```diff
@@ -54,6 +54,7 @@ TMainConsole::TMainConsole(Host* pH, QWidget* parent)
 , mSpellDic()
 , mpHunspell_system(nullptr)
 , mpHunspell_shared(nullptr)
+, mpHunspellCodec_system(nullptr)
 , mpHunspell_profile(nullptr)
 , mLogFileName(QString(""))
```

Comment:
FTR though it is not within the scope of this PR this should have been left out altogether.

The default constructor for a `QString` is fine and would have happened automatically and the use of `QString("")` is pointless and non-optimum here.